test/unit/helpers-test.js

```diff
@@ -16,16 +16,5 @@ describe('helpers', function() {
         assert.equal(tester[0], 'hello');
         assert.equal(tester[1], 'hi');
     });
-
-    it('log will not thow if console is cleared', function() {
-        var tmpConsole = window.console;
-
-        window.console = null;
-        // this should not break
-        utils.log('testing');
-
-        // restore actual window console
-        window.console = tmpConsole;
-    });
```

Comment:
The test was there to make sure "log will not thow if console is cleared", not that it "will not log". We wanted the implementation of `utils.log` to guard from throwing during player code execution if `console.log` is tampered with.

The current implementation does this by  making sure `console.log` is a function before aliasing it and if not assigning a noop.

```js
export const log = (typeof console.log === 'function') ? console.log.bind(console) : function() {};
```
https://github.com/jwplayer/jwplayer/blob/v8.0.0-alpha.2/src/js/utils/helpers.js#L17

If the test were not here, I might have just done this `log = console.log.bind(console)` and that would cause this test to fail.